public/kibi_timeline.js

```diff
@@ -80,6 +81,23 @@ define(function (require) {
         if (!timeline) {
           // create a new one
           timeline = new vis.Timeline($element[0]);
+          var timezone = config.get('dateFormat:tz');
+          if (timezone !== 'Browser') {
+            var utcOffset = null;
+            if (timezone.startsWith('Etc/GMT')) {
+              // with this line we are converting 'Etc/GMT +/- [hours] into '+/- [hours]:00'
+              // Example: from 'Etc/GMT+4' into '+4:00'. Another one: 'Etc/GMT-2' into '-2:00'
+              // with timezone.charAt(7) we catch '+' or '-' and with ('0' + timezone.substring(8)).slice(-2)
+              // we get number of hours and, when we have hours with only one digit, we add 0 before
+              // Example: '4' will be transform into '04:00'
+              utcOffset = timezone.charAt(7) + ('0' + timezone.substring(8)).slice(-2) + ':00';
```

Comment:
@giorgiocasati what I was not sure about is that it correctly works with any time shift 
+- hour > 9 or < 10 minutes 00 or 30 or 45 
See my other comment please add the tests and it will be clear that it always converts between this formats correctly